signature/jws/envelope_test.go

```diff
@@ -57,6 +45,7 @@ func getSignReq() (*signature.SignRequest, error) {
 		Expiry:                   time.Now().Add(time.Hour),
 		ExtendedSignedAttributes: nil,
 		SigningAgent:             "Notation/1.0.0",
+		SigningScheme:            "notary.x509",
```

Comment:
we can pass signing scheme as an argument of getSignReq(), so that we can use a loop to test both SigningSchemes.

Reply:
Good idea. updated.